app/angular/package.json

```diff
@@ -46,9 +49,12 @@
     "@angular/forms": ">=6.0.0",
     "@angular/platform-browser": ">=6.0.0",
     "@angular/platform-browser-dynamic": ">=6.0.0",
+    "@angular-devkit/build-angular": ">=6.0.0",
     "autoprefixer": "^8.1.0",
     "babel-loader": "^7.0.0 || ^8.0.0",
     "rxjs": "^6.0.0",
+    "typescript": "^3.4.4",
```

Comment:
Do we REQUIRE the .4 patch version? if not, try to keep dependencies to `.0` patch version for maximum compatibility.

Our lockfile will make sure we'll actually use the latest patch.

If we keep these version rounded down to the lowest patch, it's less likely users will end up with multiple versions installed.

Reply:
Done